### PR TITLE
Add ability to run linter

### PR DESCRIPTION
**Pull request by PaulChinkwende**
JIRA: https://jira.dev.bbc.co.uk/browse/NEWSART-538

Added ESLint as a dependency and then add an NPM script to execute

- [x] JIRA ticket added
- [x] Pull request URL added to JIRA ticket
- [x] Up-to-date with latest - `git pull --rebase origin latest`
- [x] Runs locally - `npm run dev` & [http://localhost:3000/](http://localhost:3000/)
- [ ] Tests added for new features
- [x] Tests pass - `npm test`

- [ ] Test engineer approval

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
   "scripts": {
     "dev": "razzle start",
     "build": "razzle build",
+    "eslint": "eslint --ext .js,jsx ./src --fix",
```
Comment (jtart) [inline package.json]: Comments:
- Can this script be called something generic? i.e. `lint`.
- We shouldn't automatically fix errors with `--fix`.
- Can linting be ran against more files than those inside `./src`?
- Can linting be ran against more than just `js` and jsx` files?

**Review by jtart — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (PaulChinkwende) [reply package.json]: It can but I thought these are the only files/file types we currently want to run linting on ?

File: package.json [hunk 2]
```diff
@@ -41,8 +45,7 @@
   "devDependencies": {
     "babel-eslint": "^7.2.3",
     "babel-loader": "^7.1.4",
-    "enzyme": "^3.3.0",
-    "enzyme-adapter-react-16": "^1.1.1"
+    "eslint-plugin-react": "^7.8.2",
```
Comment (sareh) [inline package.json]: Did you mean to add this as a dev dependency?

File: package.json (hunk 1 above)
Comment (jtart) [reply package.json]: Okay, should be fine for now. We can change down the road if needed.

**Review by jtart — DISMISSED** [review]

File: package.json (hunk 1 above)
Comment (PaulChinkwende) [reply package.json]: nice one

File: package.json (hunk 2 above)
Comment (PaulChinkwende) [reply package.json]: No, I have removed this now

File: src/client/index.jsx [hunk 3]
```diff
@@ -12,8 +12,7 @@ ensureReady(routes).then(data =>
       <After data={data} routes={routes} />
     </BrowserRouter>,
     root,
-  ),
-);
+  ));
```
Comment (pjlee11) [inline src/client/index.jsx]: should this of changed?

**Review by pjlee11 — APPROVED** [review]
LGTM 👍

**Review by jtart — APPROVED** [review]

**Pull request merged**